### ISSUE TITLE
[Internal] rehearsal migrate, when ts-config.json has a comment fails parsing

### ISSUE THREAD

**Issue by wondersloth** [rehearsal-js/rehearsal-js#876]
When a `ts-config.json` exists in a project and has a comment, it breaks a json parser in rehearsal. We need this to support comments for internal products.

```
{
  "$schema": "https://json.schemastore.org/tsconfig",
  "extends": "@linkedin/ts-bases/node16/tsconfig.json",
  "compilerOptions": {
    "allowJs": true
  }
  /**

   You will need to add relative paths in the "include" options that
   point to the root of your source and tests.

  "include": [
    //
    // For example:
    //
    // "src",
    // "test"
  ]

  */
}
```
This happening after:

```
  rehearsal:migration-graph-sahred:PackageGraph       couldNotResolve: false,
  rehearsal:migration-graph-sahred:PackageGraph       dependencyTypes: [Array],
  rehearsal:migration-graph-sahred:PackageGraph       matchesDoNotFollow: false,
  rehearsal:migration-graph-sahred:PackageGraph       circular: false,
  rehearsal:migration-graph-sahred:PackageGraph       valid: true
  rehearsal:migration-graph-sahred:PackageGraph     }
  rehearsal:migration-graph-sahred:PackageGraph   ],
  rehearsal:migration-graph-sahred:PackageGraph   dependents: [],
  rehearsal:migration-graph-sahred:PackageGraph   orphan: false,
  rehearsal:migration-graph-sahred:PackageGraph   valid: true
  rehearsal:migration-graph-sahred:PackageGraph } +1ms
  rehearsal:migrate:analyze migrationOrder [
✔ Validate project
✔ Initialize
↓ Install dependencies [SKIPPED]
↓ Create tsconfig.json [SKIPPED]
↓ Create eslint config [SKIPPED]
✔ Add package scripts
✖ /private/var/folders/9n/72xhx63x5gzcqchlbd5skvz40003jl/T/reheasal-smoke_-27421-IDaiBHi2uUMH/urn-utils/tsc…
◼ Convert JS files to TS
error:   SyntaxError: /private/var/folders/9n/72xhx63x5gzcqchlbd5skvz40003jl/T/reheasal-smoke_-27421-IDaiBHi2uUMH/urn-utils/tsconfig.json: Unexpected token / in JSON at position 161
```

**Comment by lynchbomb** [comment]
Ok we need to use json5 for parsing then

**Comment by lynchbomb** [comment]
Fixed with #883
